contributors/guide/reviewer-guidelines.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Code Review Guidelines"
+weight: 1
+slug: "reviewer-guidelines" 
+---
+
+*This Section is new and in progress. Expect this document to change often*
+
+**Table Of Contents**
+
+- [Reviewer Guidelines](#reviewer-guidelines)
+    + [Reviewing Process](#reviewing-process)
+    + [Reviewing Ideology](#reviewing-ideology)
+    + [Something To Avoid](#something-to-avoid)
+    + [Additional Readings](#additional-readings)
+
+## Reviewing Process
+
+These are some general guidelines for review, based on the size or type of pull request. 
+
+Note that each reviewer has their own style, adapted over time, so don't feel you need to explicitly commit to every one of these guidelines.
+
+### Flakes/Test Fixes
+- If the PR is to fix any test in the repository's CI , you can easily review the change by seeing the `Green Ticks` in the PR Checks.
```

Comment:
if you're going to keep this section, please remove "easily", it's not easy if it's your first time reviewing. :)